tsconfig.json

```diff
@@ -18,5 +18,6 @@
       "resolveJsonModule": true
     },
     "exclude": ["dist", "node_modules"],
-    "include": ["./src/index.ts", "./types"]
+    "include": ["./src/index.ts", "./types"],
+    "files": ["./hardhat.config.ts"]
```

Comment:
This change was required for the ts compiler to pick up imports in the hardhat config that extend the hardhat runtime and update types accordingly

Reply:
This wasn't enough, so I created https://github.com/safe-global/safe-contracts/pull/573